Update recipe - fixes #31

sdymj84 opened the pull request.

Commit:
Initial commit

Commit:
Add initial DB/API

Commit:
Update to separate folder

Commit:
test server

Commit:
test server

Commit:
test

Commit:
test

Commit:
Update foldering for separate hosting

sdymj84 committed:
Update foldering for separate hosting (#1)

* Add initial DB/API

* Update to separate folder

* test server

* test

* Update foldering for separate hosting

Commit:
Add dev/start script on package.json

sdymj84 committed:
Add signup api - in progress

sdymj84 committed:
Merge branch 'master' into dev

Commit:
Add firebase auth

Commit:
Update signup api and doc

Commit:
save

sdymj84 committed:
Add test for signup

sdymj84 committed:
Merge branch 'dev' into dev

sdymj84 committed:
Add signin api / test

Commit:
working on user info update api

sdymj84 committed:
Add update userinfo api and refactor code

Commit:
Update backend readme for schedule

Commit:
Merge branch 'dev' of https://github.com/Project-MKB/project-mkb into dev

Commit:
Update readme

Commit:
Update readme

Commit:
Update readme

Commit:
Update readme

Commit:
Merge branch 'dev' of https://github.com/sdymj84/project-mkb into dev

Commit:
Merge branch 'dev' of https://github.com/Project-MKB/project-mkb into dev

Commit:
Update with upstream dev

Commit:
Merge upstream dev into local dev to sync up

Commit:
Add createRecipe.js and redux action/reducers for creating recipe

Commit:
Pull heroku master

Commit:
Merge branch 'dev' of https://github.com/Project-MKB/project-mkb into dev

Commit:
Merge branch 'dev' of https://github.com/Project-MKB/project-mkb into dev

Commit:
Add heroku deploy instruction to backend/README.md

Commit:
Merge branch 'dev' of https://github.com/Project-MKB/project-mkb into dev

Commit:
Add Procfile for Heroku deployment

Commit:
Update Procfile and package.json for server start script

Commit:
Update package.json to update server start script

Commit:
.

Commit:
.

Commit:
.

Commit:
Update backend/readme.md to update heroku deploy instruction

Commit:
Update fbAdminConfig.js to refer secret keys from .env

Commit:
Update fbAdminConfig.js to refer secret keys from .env

Commit:
Sync up with upstream - merge main PR

Commit:
Update route to auth route for main/details

Commit:
Delete unused files

Commit:
Delete unused files

Commit:
Merge branch 'dev' into create-recipe

Commit:
.

Commit:
.

sdymj84 committed:
Delete recipes.js

sdymj84 committed:
Delete users.js

sdymj84 committed:
Delete recipes.js

sdymj84 committed:
Delete users.js

Commit:
delete unused files

Commit:
Merge branch 'dev' of https://github.com/sdymj84/project-mkb into dev

Commit:
.

Commit:
Merge branch 'dev' into create-recipe

Commit:
Merge PR - Details page (#40)

Commit:
Merge PR Details page (#40)

Commit:
Update details.js to correctly import Hero / Update RecipeInstructions.js to change class to className

Commit:
Add update recipe page and redux

kststudio reviewed (changes requested):
There is a conflict on the following files:
frontend/src/App.js
frontend/src/redux/actions/recipeActions.js
frontend/src/redux/reducers/recipeReducer.js

Commit:
Merge upstream dev - create recipe / list all recipes api

Commit:
Fix merge conflict

sdymj84 commented:
Fixed the merge conflict. Can you review again?

Commit:
Update update recipe api to inclue clone

When user trys to update the recipe that they created, it'll just update it
When user trys to update other person's recipe, it'll clone first and update the cloned recipe

Commit:
Update detail page - add button to redirect to update recipe page. fixes #46
